Source/CesiumRuntime/Private/CesiumGltfPointsComponent.h

```diff
@@ -0,0 +1,19 @@
+// Copyright 2020-2021 CesiumGS, Inc. and Contributors
```

Comment:
I just copied this from the `CesiumGltfPrimtiiveComponent` file -- should we update this for 2023?

Reply:
Haha yeah we should update it everywhere. But that can be a separate PR. It's unfortunate that Epic's marketplace guidelines require us to put this nonsense in every file.
https://www.unrealengine.com/en-US/marketplace-guidelines#262b